src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -105,6 +105,16 @@ static View()
             RegisterAccessibilityDelegate();
         }
 
+        static internal new void Preload()
+        {
+            Container.Preload();
+
+            // Do nothing. Just call for load static values.
+            var object0 = positionPropertyGroup;
```

Comment:
I know this PR is to fix the urgent VD issues.
But, is it okay for the variable name???

```suggestion
            var positionBbject = positionPropertyGroup;
```

Reply:
Thank you for your comments. Now I make them as 'temporalXXX', instead of meaningless variable name :)

Reply:
Thank you for the quick update!